elasticsearch/src/main/java/com/gentics/mesh/search/index/node/NodeIndexHandlerImpl.java

```diff
@@ -826,18 +825,22 @@ private IndexInfo createDummyIndexInfo(SchemaModel schema, String language) {
 	 * @param type
 	 * @return
 	 */
-	public String generateVersion(NodeGraphFieldContainer container, String branchUuid, ContainerType type) {
+	public String generateVersion(HibNodeFieldContainer container, String branchUuid, ContainerType type) {
 		return getTransformer().generateVersion(container, branchUuid, type);
 	}
 
 	@Override
 	public Function<String, HibNode> elementLoader() {
-		return (uuid) -> HibClassConverter.toGraph(db).index().findByUuid(Node.class, uuid);
+		// TODO override in OrientDB implementation to use index for performance, if required
```

Comment:
The OrientDB implementation has been readdressed here, due to the currently missing mechanism of dependency override:
https://github.com/gentics/mesh/pull/1277